src/legacy/ui/public/field_editor/components/field_format_editor/editors/color/color.test.js

```diff
@@ -21,7 +21,7 @@ import React from 'react';
 import { shallowWithIntl } from 'test_utils/enzyme_helpers';
 
 import { ColorFormatEditor, ColorFormatEditorComponent } from './color';
-import { DEFAULT_COLOR } from '../../../../../../../core_plugins/kibana/common/field_formats/types/color_default';
+import { DEFAULT_COLOR } from '../../../../../../../../plugins/data/common/field_formats/converters/color_default';
```

Comment:
not sure that DEFAULT_COLOR should be in converters folder. What about to move it into utils

Reply:
Moved to `constants`